src/main/java/core/basesyntax/dataprocess/DataProcessor.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.dataprocess;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.Strategy;
+import core.basesyntax.strategy.impl.BalanceService;
+import core.basesyntax.strategy.impl.PurchaseService;
+import core.basesyntax.strategy.impl.ReturnService;
+import core.basesyntax.strategy.impl.SupplyService;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class DataProcessor {
+    private final List<FruitTransaction> fruitTransactions;
+    private final Map<String, Integer> fruitData;
```

Comment:
you store this in storage, i'm not sure there's a point to have these fields in this class